src/entities/event/ui/event-card/event-card.module.scss

```diff
@@ -38,8 +38,7 @@
       line-height: 1.2;
       color: $color-gray-600;
 
-      background-color: rgb(255 255 255 / 5%);
-      border: 4px solid #cdbaeb;
+      border: 2px solid $color-purple-400;
```

Comment:
I've asked @Illia-Sakharau about it. Maybe it will be better to add new `purple` color to our constants.

Reply:
I've added a new color for the border.